recipes-kernel/linux/files/patches-5.10/0117-Revert-serial-8250-Don-t-touch-RTS-modem-control-whi.patch

```diff
@@ -0,0 +1,45 @@
+From eb4ac23600231afe73ab7b4bb8aef644b4046687 Mon Sep 17 00:00:00 2001
+From: Su Bao Cheng <baocheng.su@siemens.com>
+Date: Mon, 22 Nov 2021 15:59:01 +0800
+Subject: [PATCH 117/117] Revert "serial: 8250: Don't touch RTS modem control
+ while in rs485 mode"
+
+This reverts commit f45709df7731ad36306a28a3e1af7309d55c35f5.
+
+The mcr register is not always reflect the significant value, so
+checking against it will mislead.
+
+The direct issue is RS485/422 does not fully work on 8250 serial, due to
+the MCR[1] is not set correctly with the to-be-converted commit.
```

Comment:
The correct reasoning is that the original patch was trying to avoid user-space triggered changes but incorrectly blocked also valid kernel-originated changes because it hooked into the wrong function.

Reply:
ACK and revised.